lib/yt/video_audit/youtube_association.rb

```diff
@@ -17,8 +17,13 @@ def description
 
       def valid?(video)
         video.description.split(' ')
-             .select {|word| Yt::URL.new(word).kind == :channel }
-             .any? {|link| Yt::Channel.new(url: link).id == video.channel_id }
+          .select {|word| Yt::URL.new(word).kind == :channel }
+          .any? do |link|
+            video.channel_id == begin
+                Yt::Channel.new(url: link).id
+              rescue Yt::Errors::NoItems => e
+              end
+          end
```

Comment:
This method is becoming very hard to read.

Can you split into multiple methods? Thanks! :bow: